### PR TITLE
Add about page, fix nextlinks

### PR DESCRIPTION
**Pull request by LanesGood**
This PR:
1. Adds an About page with background on the project, references and links to Github and project supporters with logos
2. Ensures that the Next `<Link>` Component always joins the environment variable `APP_URL`  to the href target
    - _Note: Item 2 is likely required for all non-standard environments_
3. Updates styles on links, text styles, and page navigation buttons
4. Removes unused icons
5. Replaces the SVGSquare component with the Badge component

image

### CODE REVIEW

**Comment by LanesGood** [comment]
@kamicut please feel free to add to/edit/contribute content for the about page in addition to review

**Comment by kamicut** [comment]
@LanesGood thanks for the PR! In NextJS, we use the `basePath` to ensure that Links are correctly formed https://nextjs.org/docs/api-reference/next.config.js/basepath, so we don't need to join the `APP_URL` with the link path. Is this required for a specific reason in your commit?

**Comment by LanesGood** [comment]
@kamicut I was simply following the pattern established on other NextLinks throughout the app.

**Comment by LanesGood** [comment]
@kamicut per our conversation, I've removed all instances of joined APP_URL's from any Nextjs `<Link> href` in the latest commit.

File: src/pages/developers.js [hunk 1]
```diff
@@ -1,43 +1,56 @@
...
+                  href='https://github.com/developmentseed/osm-teams-node-example'
+                  target='_blank'
+                  rel='noreferrer'
+                >
+                  Example Node Integration
+                </a>
+              </ListItem>
+              <ListItem>
+                <a
+                  href='https://github.com/thadk/osm-teams-python-example'
```
Comment (kamicut) [inline src/pages/developers.js:43]:
```suggestion
                  href='https://github.com/developmentseed/osm-teams-python-example'
```

**Pull request merged**